packages/devextreme/js/__internal/ui/html_editor/ui/aiDialog.ts

```diff
@@ -719,6 +724,16 @@ export default class AIDialog extends BaseDialog<AIDialogResult> {
     return visible as boolean;
   }
 
+  private _refreshDialogAria(): void {
+    const id = String(new Guid());
+    // @ts-expect-error $overlayContent does not exist on public type
+    const $overlayContent = this._popup.$overlayContent();
```

Comment:
There is a guide in out public docs how to do it without private API, please check

Reply:
Did someone say that it's equal?

Reply:
`$overlayContent = $(e.component.content()).parent()`
https://js.devexpress.com/jQuery/Documentation/Guide/UI_Components/Popup/Accessibility/#Screen_Reader_Support

Reply:
[77b8b464c86180d4cd32fbb7e26059eb6288691b]